### ISSUE TITLE
Style guide question regarding capitalization of headers

### ISSUE THREAD

**Issue by kytrinyx** [exercism/docs#389]
I think I recall that headers on Exercism should capitalize only the first letter and proper names, but I can't find a reference to this.

Have we documented our header preferences anywhere? If not, should we add something to `docs/building/markdown/style-guide` about it?

**Comment by ErikSchierboom** [comment]
I don't think we do, and I'm not even sure we're consistent :( I know Jeremy sometimes likes to capitalize multiple words in sentence. I'm personally of the "capitalize the first letter only" camp 🤷

**Comment by kytrinyx** [comment]
I'm also (quite strongly) in the "capitalize the first letter only" camp. I wonder if it's a Dutch/Norwegian vs English thing.
